src/app/booking/OOSPlannerSection.js

```diff
@@ -225,6 +273,51 @@ function useAwaitingPaidStatusBill(billIdInitial = null) {
   const { setBillId } = useBillPolling({ billIdInitial, onSuccess });
 }
 
+function FriendsBookings( {children, weekDates, bookings} ) {
+  const [selectedChild, setSelectedChild] = useState(children[0]);
+  return (
+    <>
+      <TableRow>
+        <StyledTableCell rowSpan={2} sx={{ verticalAlign: "bottom", textAlign: "center" }}>
+          <PersonAddAltIcon />
+          <Typography sx={{ fontWeight: 700 }}>Friends</Typography>
+        </StyledTableCell>
+      </TableRow>
+      <TableRow>
+      <StyledTableCell colSpan={5}>
+      <Box sx={{ display: "flex", alignItems: "center", gap: 1 }}>
+        {children.map((child) => (
+          <Button key={child.id} variant={ selectedChild === child ? "contained" : "outlined"} color="grey" onClick={() => setSelectedChild(child)}>
```

Comment:
What if I want to see bookings for all my children friends?
See how am / pm for activities list implemented.

Reply:
But do we need it? I asked